test/constants.js

```diff
@@ -4,7 +4,8 @@ const constants = {
 		RAF_THRESHOLD : 16,
 		SMALL: 5
 	},
-	ITEM_TO_OBSERVE: 5
+	ITEM_TO_OBSERVE: 5,
+	SKIP_CYCLE: 3
```

Comment:
Should I change the name consistent with src/engine.ts constant? I name it as NUM_SKIPPED_FRAMES to avoid confusion?